packages/logger/package.json

```diff
@@ -28,16 +28,14 @@
   "license": "MIT",
   "dependencies": {
     "fluent-logger": "^3.0.0",
-    "winston": "^3.0.0"
+    "winston": "^3.1.0"
```

Comment:
this is still in alpha - will this be an issue? (ref: https://github.com/winstonjs/winston/blob/master/CHANGELOG.md)